internal/transport/transporter.go

```diff
@@ -10,4 +12,6 @@ type Transporter interface {
 	Disconnect(quiesce uint)
 	SendData(data []byte, dest string) error
 	ReceiveData(data []byte, dest string) error
+	// Reload reloads the transport with the given tls.Config
```

Comment:
We didn't document any of the other interface methods. For consistency, we should either document all of them or none of them.